routing/routing.go

```diff
@@ -369,8 +386,17 @@ func (r *Routing) startReceivingUpdates(o Options) {
 						r.firstLoadSignaled = true
 					}
 				}
+
+				close(goodEndpointerQuit)
+				timer.Stop()
+				goodEndpointerQuit = make(chan struct{})
+				timer = time.NewTicker(500 * time.Millisecond)
```

Comment:
Why do we need to pass timer to GoodEndpointerGoroutine, i.e. why not create it inside?

Reply:
Why is GoodEndpointerGoroutine exported?